src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $cell-size * 7 + $gap + 10px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @include daySize($cell-size);
+
+    background-color: #eee;
+    border: 1px solid #000;
+    box-sizing: border-box;
+
+    &:hover {
+      background-color: #eea3a3;
+
+      transform: translate(0, -20px);
+      transition: background-color 0.5s, transform 0.5s;
```

Comment:
![image](https://user-images.githubusercontent.com/112480002/194115999-a304ee50-4bf2-4208-841a-c7e23cf30bbc.png)
9 point in checklist , remove it from :hover, and move to &__day.